### PR TITLE
Do not include docker dir in tarball export of DTK

### PR DESCRIPTION
**Pull request by dalg24**
Related to #133

### CODE REVIEW

**Comment by Rombur** [comment]
We don't want anything from `docker` in the tarball? I would just have removed `travis` and `jenkins`

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+docker/ export-ignore
+.clang* export-ignore
+.ycm_extra_conf.py export-ignore
+.travis.yml export-ignore
```
Comment (Rombur) [inline .gitattributes]: Add .mailmap

**Comment by dalg24** [comment]
> We don't want anything from docker in the tarball? I would just have removed travis and jenkins

I would just remove the whole directory.  If we only remove `jenkins/` and `travis/` sub dirs we need to update the image tag in the docker compose file and guarantee that it will always be available for download.  Also nvidia extension utility for docker-compose is already getting outdated now that nvidia-docker has been around for a while.

File: .gitattributes (hunk 1 above)
Comment (aprokop) [reply .gitattributes]: Do we still need `.project` and `.cproject`?

Comment (dalg24) [reply .gitattributes]: They are unnecessary

**Comment by Rombur** [comment]
@dalg24 I still think that having a docker images of the release is useful for people who just want to see how DTK works but that can be a step in #133

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+.mailmap export-ignore
```
Comment (Rombur) [inline .gitattributes]: You can also ignore `.gitattributes`

**Pull request merged**